Scenes/InformationPopup/Popup.gd

```diff
@@ -0,0 +1,42 @@
+extends Control
+
+export(float) var mouse_y_offset
+
+var mouse_pos : Vector2
+
+onready var label = find_node("Label")
+	
+func init(Title:String):
```

Comment:
top-down design rule...
`init`, `set_title`, `update_size_and_pos`, `update_size`, `update_position` ---> low and lower levels of abstraction

Reply:
Right. I moved `_input` function above the `update_size`. Do You think it's OK?